src/AcmLabels/AcmLabels.tsx

```diff
@@ -54,40 +49,20 @@ export function AcmLabels(props: {
     /* istanbul ignore next */
     const expandedText = props.expandedText ?? 'Show less'
 
-    const onClick = useCallback(() => setShowMore((showMore) => !showMore), [])
-
-    /* istanbul ignore next */
-    const onKeyPress = useCallback((e) => {
-        if (e.key === 'Enter' || e.key === ' ') {
-            setShowMore(!showMore)
-            e.preventDefault()
-        }
-    }, [])
+    if (props.labels === undefined) return <Fragment />
 
     return (
-        <span className="acm-labels">
+        <LabelGroup numLabels={labels.length} expandedText={expandedText} collapsedText={collapsedText}>
             {labels.map((label) => (
-                <Tooltip key={label} content={label}>
-                    <span className="acm-label">{label}</span>
-                </Tooltip>
+                <Label key={label}>
+                    <Truncate content={label} />
+                </Label>
```

Comment:
@jamestalton Great to use components, but Truncate is in beta and for some reason it sets a minimum width.
<img width="886" alt="image" src="https://user-images.githubusercontent.com/42188127/172887835-915f65b2-17fc-4651-8b30-db17978b7fe2.png">

<img width="189" alt="image" src="https://user-images.githubusercontent.com/42188127/172887971-df8dcd29-32b2-4b15-bd5c-242df327200d.png">

Seems strange that a component that aims to save space would use a minimum amount. Do you think we should override that variable and set it to 0? Or we can ask why the mid width for Truncate.

Reply:
Had to use the Truncate to solve the problem in ACM details pages where labels overflowed the details column.

Reply:
![Screen Shot 2022-06-09 at 12 34 43 PM](https://user-images.githubusercontent.com/6277895/172898731-a7f263ba-2ffb-4b25-9e4b-bb0d3f2a609e.png)

Reply:
The minimum of 0 would work.
There is an internal span that has a second minimum that we cannot access easily.

Reply:
![Screen Shot 2022-06-09 at 12 40 16 PM](https://user-images.githubusercontent.com/6277895/172899813-4303fc52-9395-4003-8900-af7f77689fd7.png)